File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

void makeHash(void);

int main(void)
{
    int h;

//get height from user
    do
    {
        h = get_int("Height: ");
    }

//check height is between 1 and 8
    while (h > 8 || h < 1);

//draw the blocks
    for (int row = 1; row <= h; row++)
    {
        for (int space = 0; space < (h - row); space ++)
        {
            printf(" ");
        }
        for (int hash = 1; hash <= row; hash++)
        {
            makeHash();
        }

        printf("  ");

        for (int hash = 1; hash <= row; hash++)
        {
            makeHash();
        }


        printf("\n");
    }
}


//do the number of hashes

void makeHash(void)
{

    printf("#");
}

File: pset6/mario/less/mario.py
from cs50 import get_int

# get the height from user make it height variable
height = 0

while height < 1 or height > 8:
    height = get_int("Height: ")
else:
    # make some variables for drawing the hash and spaces
    hash = 1
    space = height - 1

# loop to print but only if height in correct range

    for i in range(height):
        print(" " * space, end="")
        print("#" * hash)
        space = space - 1
        hash = hash + 1